floto/templates/base.html

```diff
@@ -49,6 +49,7 @@
     <script src="{% static 'util.js' %}"></script>
     <script src="{% static 'environment_component.js' %}"></script>
     <script src="{% static 'timing_component.js' %}"></script>
+    <script src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js"></script>
```

Comment:
Similar here, I think you can add this as the first part of your scripts block in the overview template.